docs/components/emails.rst

```diff
@@ -1,13 +1,434 @@
+                $to   = $message->getTo();
+                $body = $message->getBody();
+
+                // Configuration values that were set by the user through Mautic's Configuration screen
+                $host   = $this->coreParametersHelper->get('mautic.mailer_host');
+                $apiKey = $this->coreParametersHelper->get('mautic.mailer_api_key');
+
+                // Do your magic for sending the email here
+                // $myService->send(...)
+
+                // Return the number of recipients who were accepted for delivery
+                return 1;
+            } catch (\Exception $e) {
+                $this->triggerSendError($event, $failedRecipients);
+                $message->generateId();
+                $this->throwException($e->getMessage());
+            }
+
+            // Return the number of recipients who were accepted for delivery
+            return 0;
+        }
+
+        /**
+        * @inheritdoc
+        */
+        public function getMaxBatchLimit(): int
+        {
+            return 50;
+        }
+
+        /**
+        * @inheritdoc
+        */
+        public function getBatchRecipientCount(\Swift_Message $message, $toBeAdded = 1, $type = 'to'): int
+        {
+            $toCount  = is_array($message->getTo()) ? count($message->getTo()) : 0;
+            $ccCount  = is_array($message->getCc()) ? count($message->getCc()) : 0;
+            $bccCount = is_array($message->getBcc()) ? count($message->getBcc()) : 0;
+
+            return null === $this->batchRecipientCount ? $this->batchRecipientCount : $toCount + $ccCount + $bccCount + $toBeAdded;
+        }
+
+        /**
+        * @inheritdoc
+        */
+        public function getCallbackPath(): string
+        {
+            return 'helloworld_api';
+        }
+
+        /**
+        * @inheritdoc
+        */
+        public function processCallbackRequest(Request $request)
+        {
+            $postData = json_decode($request->getContent(), true);
+
+            // Handle the callback here
+        }
+
+        private function triggerSendError(\Swift_Events_SendEvent $evt, array &$failedRecipients): void
+        {
+            $failedRecipients = array_merge(
+                $failedRecipients,
+                array_keys((array) $this->message->getTo()),
+                array_keys((array) $this->message->getCc()),
+                array_keys((array) $this->message->getBcc())
+            );
+
+            if ($evt) {
+                $evt->setResult(\Swift_Events_SendEvent::RESULT_FAILED);
+                $evt->setFailedRecipients($failedRecipients);
+                $this->getDispatcher()->dispatchEvent($evt, 'sendPerformed');
+            }
+        }
+    }
 
 Email stat helpers
------------------------
+------------------
 
-todo see  ``\Mautic\EmailBundle\Stats\Helper\StatHelperInterface``
\ No newline at end of file
+This section is in progress. See  ``\Mautic\EmailBundle\Stats\Helper\StatHelperInterface``
```

Comment:
Anyone who has experience with email stat helpers? Shall we just leave it out entirely for now? It wasn't in the old docs.